### PR TITLE
Playground itinerary

### LINKED ISSUE #3: Create WRPlayground itinerary for new members
# Rationale
Simplify the training process and address on of the main issues in software attrition: the training program is really long and not necessarily relevant to skills required on the WRover

# Required Actions
- [x] Proposed itinerary contains training on ROS setup
- [x] Proposed itinerary contains training on ROS Topics
- [x] Proposed itinerary contains training on ROS Services
- [x] Proposed itinerary contains training on ROS debugging/tooling
- [x] Proposed itinerary has a timeline mapping specific concepts to meetings
- [x] Itinerary is documented in the repository

### PR DESCRIPTION
**Pull request by Tzanccc**
Itinerary for WRoverPlayground, a tutorial project for new students to quickly get into ROS and the team's code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# WRoverPlayground
+Welcome to WRoverPlayground! These learning modules will help you become somewhat proficient in using the Robot Operating System (ROS). Each module builds on the previous ones. When you are finished, you will have built a robot named WRunt. This robot will be able to autonomously navigate through a course of infrared (IR) beacons from a starting location. *Add a picture of the course for illustrative purposes. A GIF of WRunt completing a course would be even better*
```
Comment (bennowotny) [inline README.md]: Not necessarily 'build the robot', but will be compatible with said robot.

File: README.md
Comment (bennowotny) [inline README.md:1]: Maybe describe the challenge in flowchart-fashion to give a clearer idea of how the challenge flow works.

File: PlanningDoc.md
Comment (bennowotny) [inline PlanningDoc.md:1]: Order may change to pull ROS debug tools first so students can use said tools to test their code as they write it.

File: PlanningDoc.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### ROS debugging/tooling
+* Using rviz to visualize robot and beacon position
+* Using rqt_plot to monitor topic data
+
+## Steps and instructions
+
+Ideally each step should take one meeting session, so studenets should be able to complete the playground tutorial by the end of the second week. 
+
+### Step 1: Development environment and ROS setup
```
Comment (bennowotny) [inline PlanningDoc.md:32]: Tooling may also move in here so students can use said tools to debug the future code while they're writing it.

**Review by bennowotny — CHANGES_REQUESTED** [review]

File: PlanningDoc.md
Comment (bennowotny) [reply PlanningDoc.md:1]: RESOLVED: Outdated

File: README.md (hunk 1 above)
Comment (bennowotny) [reply README.md]: Addressed f8c0c00.

File: PlanningDoc.md (hunk 2 above)
Comment (bennowotny) [reply PlanningDoc.md:32]: Not req'd to address.

File: README.md
Comment (Tzanccc) [reply README.md:1]: Added in [f16371a](https://github.com/WisconsinRobotics/WRoverPlayground/pull/8/commits/f16371a0f8886efccb3544783daf10e40340e5bf)

File: PlanningDoc.md (hunk 2 above)
Comment (Tzanccc) [reply PlanningDoc.md:32]: Added in [f16371a](https://github.com/WisconsinRobotics/WRoverPlayground/pull/8/commits/f16371a0f8886efccb3544783daf10e40340e5bf)

File: README.md
Comment (bennowotny) [reply README.md:1]: Addressed in 14554aa1bc91c.

**Review by bennowotny — APPROVED** [review]

**Pull request merged**